### ISSUE TITLE
Allow multiple archive nodes under an Articulate root

### ISSUE THREAD

**Issue by Shazwazza** [Shazwazza/Articulate#9]
see: http://our.umbraco.org/projects/starter-kits/articulate/discussions/54492-Enable-multiple-archives

**Comment by lele7** [comment]
Hi!

Do you plan to implement this issue soon?

**Comment by Shazwazza** [comment]
I'm pretty pressed for time, i'd assume i won't have time to look at this until late June.

**Comment by lele7** [comment]
Any news or release date on this enhancemnet?

**Comment by Shazwazza** [comment]
Sorry, i'm far too busy at the moment to have a look at this so cannot give you any estimates. I plan on releasing a new Articulate version soon which has quite a lot of other fixes: https://github.com/Shazwazza/Articulate/milestones/1.1.0

but this one is a more in-depth fix/change

**Comment by Shazwazza** [comment]
Ok I actually got this working :)

See the above commit for the changes. It was actually quite easy to do and I've tested that the latest blog posts show, that additional archive posts show up in rss and in tags, etc... So it should 'just work'. 

Some caveats however, currently for LiveWriter support and the mobile editor it will only post to the first archive node found, similarly importing blogml will be the same.
